azkaban-web-server/src/main/java/azkaban/webapp/servlet/UserUtils.java

```diff
@@ -0,0 +1,27 @@
+package azkaban.webapp.servlet;
+
+import azkaban.user.Permission;
+import azkaban.user.Role;
+import azkaban.user.User;
+import azkaban.user.UserManager;
+import azkaban.webapp.AzkabanWebServer;
+
+
+public final class UserUtils {
+  private UserUtils() {
+
+  }
+
+  public static boolean
+  hasPermissionforAction(final UserManager userManager, final User user, final Permission.Type type) {
```

Comment:
We have Guice support now. Consider using Guice to manage the dependency on userManager.